model/src/main/kotlin/config/PathExcludeReason.kt

```diff
@@ -62,6 +62,12 @@ enum class PathExcludeReason {
      */
     PROVIDED_BY,
 
+    /**
+     * The path only contains code or tools used for testing source code which are not included in distributed build
```

Comment:
I wonder if it would make sense to make it a bit more generic by saying:

`The files within path are used primarily for testing and none of the files gets included in the distributed artifact.`

...This relaxes the `only` to `mostly` which can make sense as the classification might be useful if it only applied to 98% of the files underneath path while it is still strict about the fact that `none` of the files must go into the release.
Further there might be files used for testing only which do not fit into the categories `code` and `tools`. E.g. test assets, data files.

Reply:
Discussed F2F with @fviernau  and made changes.